api/stse_data_storage.c

```diff
@@ -116,10 +123,10 @@ stse_ReturnCode_t stse_data_storage_update_zone(
 	stsafea_update_option_t options;
 
 	options.atomicity = atomicity;
-	options.change_ac_indicator = 0 ;
+	options.change_ac_indicator = (stsafea_zone_ac_change_indicator_t)0 ;
```

Comment:
Please avoid casting a constant value when the casted type is an enum, use this enum values instead.

_This comment is applicable for all cast in this file_